ui/src/app/dashboard/dashboard.component.html

```diff
@@ -1,18 +1,18 @@
 <jm-header [showControls]="false"></jm-header>
 
 <!--TODO(zachx): improve the grid layout-->
-<mat-grid-list cols="6" rowHeight="200px">
-  <mat-grid-tile
-    [colspan]="3"
-    [rowspan]="2">
+<!--<mat-grid-list cols="6" rowHeight="200px">-->
```

Comment:
What are these changes? Does this still render?

Please just remove code instead of commenting it out.

Reply:
Removed